misc/helper-scripts/taste-create-data-view

```diff
@@ -97,7 +117,60 @@ fi
 
 # cp $(taste-config --prefix)/share/taste-types/taste-types.asn .
 
-taste-update-data-view
+# If ROS messages will be generated
+if [ $GEN_ROS_MSGS ]
+then
+    # Generate TASTE extended types
+    echo 'TASTE-ExtendedTypes DEFINITIONS ::=
+    BEGIN
+
+    IMPORTS
+    T-Int32, T-UInt32, T-Int8, T-UInt8, T-Boolean  FROM TASTE-BasicTypes;
+
+    -- Set of TASTE predefined basic types
+
+    T-Double ::= REAL (-1.7976931348623157e+308..1.7976931348623157E308)
+
+    T-Float ::= REAL (-3.402823466E+38..3.402823466e+38)
+
+    T-Int16 ::= INTEGER (-32768..32767 )
+
+    T-UInt16 ::= INTEGER (0 .. 65535 )
+
+    T-UInt64 ::= INTEGER (0..9223372036854775807)
+
+    T-Int64 ::= INTEGER (-9223372036854775808.. 9223372036854775807)
+
+    -- Maximum string size
+    maxSize-T-String T-UInt32 ::= 256
+
+    T-String ::= OCTET STRING (SIZE(0..maxSize-T-String))
+
+    END' > taste-extended-types.asn
+
+    echo 'Time-Types DEFINITIONS ::=
+    BEGIN
+    IMPORTS T-UInt32 FROM TASTE-BasicTypes;
+    Time   ::=
+    SEQUENCE
+    {
+    sec             T-UInt32,
+    nsec            T-UInt32
+    }
+    END' > time-types.asn
```

Comment:
In contrast to the auto-generated messages above these seem to lack a little in formatting.

Reply:
Done